### DISCUSSION TITLE
how to create react component library using wmr and typescript?

### DISCUSSION THREAD

**Discussion by leamandeep** [preactjs/wmr#960] [Q&A] [answered]
Before jumping into the core question. I wanna ask `is wmr maintained?`. 
If maintained, can you guide me on creating react component library using wmr?

**Comment by rschristian** [comment, marked as answer]
It's not actively maintained anymore, no. It still works and fulfills some use cases really well, but no one is dedicating time to it.

WMR is a build tool for Preact apps. It wouldn't help you build a React component library.

**Reply by leamandeep** [reply]
thanks for quick replying

**Reply by rschristian** [reply]
Might be worth taking a look at [`microbundle`](https://github.com/developit/microbundle) for fulfilling that though.

**Reply by rschristian** [reply]
It's a build tool that's targeted at libraries/modules. Certainly can use it to build a React component lib.

**Reply by leamandeep** [reply]
Again Thanks
